apps/console/src/features/organizations/models/index.ts

```diff
@@ -18,3 +18,5 @@
 
 export * from "./endpoints";
 export * from "./organizations";
+export * from "./permission";
+
```

Comment:
Do we need this empty line?

Reply:
Removed in https://github.com/wso2/identity-apps/pull/3150/commits/4881fb75f0c759b0224acf6e192dc4addb2e9cf4